ci-operator/step-registry/aws-load-balancer/tag-vpc/aws-load-balancer-tag-vpc-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: aws-load-balancer-tag-vpc
+  from_image:
+    namespace: ocp
+    name: "4.12"
```

Comment:
4.13?

Reply:
Took the same version as from the pre step I used: `ipi-aws-pre-proxy`. I don't think it really matters as we only use the`aws` client here.